Formula/mlt.rb

```diff
@@ -12,19 +12,31 @@ class Mlt < Formula
 
   depends_on "pkg-config" => :build
   depends_on "ffmpeg"
+  depends_on "fftw" # fft, lightshow, dance
   depends_on "frei0r"
   depends_on "libdv"
+  depends_on "libexif" # qt module needs libexif
   depends_on "libsamplerate"
   depends_on "libvorbis"
-  depends_on "sdl"
+  # mlt >= v6.21.0: depends_on "gdk-pixbuf"
+  # mlt >= v6.21.0: depends_on "pango"
```

Comment:
when will 6.21.0 be released?

Reply:
@SMillerDev, Master is already at 6.21.0 which is what I initially tried.

You'll have to ask the mlt framework guys for when they intend to officially release.

Reply:
You should either wait for the release or delete the comments then

Reply:
Why? Bad form to have comments like this or something?

Goal is to make it easier for the next person who looks at the file.

Reply:
We should only have comments that comment on the code, not to have commented out code for later.